src/js/components/productSupplier/create/styles.scss

```diff
@@ -75,3 +75,8 @@
     font-weight: 400;
   }
 }
+
+.attributes-section {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
```

Comment:
wouldn't class `form-grid-3` work for you? It's the same as this one, with only difference that is `gap: 8px`